meerkat_nest/config.py

```diff
@@ -31,6 +35,22 @@
 spec.loader.exec_module(country_config_module)
 country_config = country_config_module.country_config
 
+salt_file = os.environ.get("SALT", None)
+salt = None
+
+if salt_file:
+    try:
+        f = open(salt_file, 'r')
```

Comment:
you're not closing the file after you're done. Better use `with open(salt_filename, 'r') as file:`
```python
>>> with open('workfile') as f:
...     read_data = f.read()
>>> f.closed
True
```
from doc: https://docs.python.org/3/tutorial/inputoutput.html#reading-and-writing-files